composer.json

```diff
@@ -65,10 +68,21 @@
         ],
         "post-install-cmd": [
             "DrupalProject\\composer\\ScriptHandler::createRequiredFiles",
-            "@drupal-scaffold"
+            "@drupal-scaffold",
+            "@kalastatic"
```

Comment:
On a second thought, may be good to use `npm install` instead?  Make it just use `npm` to manage whatevs in there. Is there a way to say....

"Run `npm install` if `npm` is available."?